spec/filters/dissect_spec.rb

```diff
@@ -3,6 +3,31 @@
 require "logstash/filters/dissect"
 
 describe LogStash::Filters::Dissect do
+  class LoggerMock
```

Comment:
As an alternative, you could subscribe Cabin::Channel to a Queue or Array since they both support `<<` method, then just make assertions about the contents of the queue/array.

```ruby
let(:logs) { Array.new }
let(:loggr) { Cabin::Channel.new.tap { |c| c.subscribe(logs) } }

# `logs` will contain all messages logged to `loggr`
```